Sources/ClientRuntime/EventStream/Header.swift

```diff
@@ -34,8 +34,8 @@ extension EventStream {
         case bool(Bool)
         case byte(Int8)
         case int16(Int16)
-        case int32(Int32)
-        case int64(Int64)
```

Comment:
short - Int16
int - ..
long - ...

Reply:
Just noting here, I think where we left off in our chat is that we want to keep the explicit types here as is to avoid any casting issues.

Reply:
right we are going to use the explicit types than depending on the Swift runtime behavior. Because runtime behavior can lead to issues on the service side which has specified type for each date type.